packages/dhis-api/src/DhisApi.js

```diff
@@ -240,8 +240,8 @@ export class DhisApi {
     const queryParameters = {
       program: resolvedProgramId,
       orgUnit: resolvedOrganisationUnitId,
-      programIdScheme: 'code',
-      orgUnitIdScheme,
+      programIdScheme: 'uid',
+      orgUnitIdScheme: 'uid',
```

Comment:
Actually I don't think this change is safe. The deprecated functions are still used in a number of places (we have https://github.com/beyondessential/tupaia-backlog/issues/451 to get rid of it), and they'll be assuming the response has codes for org units and programs. Sorry @IgorNadj, I didn't look into it properly when we were talking earlier!

Reply:
Ah, no worries, reverted.